upgrades/schema/Version_5_0_20200220092617_create_read_product.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Pim\Upgrade\Schema;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * @author    Willy Mesnage <willy.mesnage@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class Version_5_0_20200220092617_create_read_product extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        $query = <<<SQL
+CREATE TABLE IF NOT EXISTS akeneo_connectivity_connection_read_product(
```

Comment:
```suggestion
CREATE TABLE akeneo_connectivity_connection_audit read_product(
```
It should break if it already exists